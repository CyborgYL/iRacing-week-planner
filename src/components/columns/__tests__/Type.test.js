import React from 'react';
import { shallow } from 'enzyme';
import { describe, test } from '@jest/globals';

import { Type } from '../index';

describe('components/columns/Type', () => {
  test('renders correctly', () => {
    const component = shallow(
      <Type
        race={{
          type: 'RX',
        }}
      />
    );

    expect(component).toMatchSnapshot();
    expect(component.text()).toBe('RX');
  });
});
