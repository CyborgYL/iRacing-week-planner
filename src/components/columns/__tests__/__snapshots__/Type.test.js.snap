// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/columns/Type renders correctly 1`] = `
<td>
  <div>
    RX
  </div>
</td>
`;
